src/libmongoc/tests/unified/operation.c

```diff
@@ -499,6 +547,7 @@ operation_bulk_write (test_t *test,
    bson_error_t op_error = {0};
 
    parser = bson_parser_new ();
+   bson_parser_allow_extra (parser, true);
```

Comment:
Was `allow_extra` actually needed here? I don't see `bson_parser_get_extra` called in this function.

Reply:
Nope! Removed.